plugin.yml

```diff
@@ -1,10 +1,10 @@
 name: PureEntitiesX
 main: revivalpmmp\pureentities\PureEntities
-version: 0.6.9
-api: [3.11.8]
+version: 0.7.0
+api: 3.19.0
 
 load: STARTUP
-authors: ["milk0417", "RevivalPMMP"]
+authors: ["milk0417", "RevivalPMMP", "supercrafter333"]
```

Comment:
Please remove your username from the authors list.  There are a large number of contributors to this project and it is preferred that people look at Github to get a full list of everyone's contributions. That is why the website is included in the plugin details.